distribution/std-lib/Standard/src/Base/Data/Any/Extensions.enso

```diff
@@ -45,61 +64,117 @@ Any.!= that = (this == that).not
 
 ## Checks if `this` is greater than `that`.
 
+   Arguments:
+   - that: The value to compare `this` against.
+
    To have `>` defined, a type must define `compare_to`, returning an Ordering.
 
+   ! Implementing Greater Than
+     Many types can admit a definition of greater than that is more efficient
+     than the generic one given here. When implementing this for your own types
+     please ensure that it is semantically equivalent to this definition.
```

Comment:
It is not clear to me what does it mean that it is semantically equivalent.

Does it mean that it is consistent with the `compare_to` method? It is not directly clear from that description.